feat: Add custom content to ConditionList widget (#3849)
kyma-project/busola: docs/extensibility/50-list-and-details-widgets.md

| **highlights**    | No       |      | A map of highlight rules that will only be applied to the `condition` matching `type`. Key refers to the type of highlight, while the rule can be a plain array of values ​​or a string containing the [JSONata](100-jsonata.md) rule. Allowed keys are `informative`, `positive`, `negative`, `critical`, and `type`. <br><br> With the `type` key (required), you can specify which condition the highlighting must be applied to. It must contain one of the `types` of the source condition. <br><br> If no highlighting is provided, the following values ​​are automatically supported: <br> - rendered as informational: `Unknown`. <br> - rendered as positive: `True`. <br> - rendered as critical: `False`. |

> | **customContent** | No       |      | Allows adding custom content to specific condition types. Each object should contain: <br> - `type` (required): The condition type to match <br> - `value` (required): A string, JSONata expression or a widget definition to render custom content <br> - `header`: Optional header text for the custom content section <br><br> The custom content will be rendered above the condition's default message.                                                                                                                                                                                                                                                                                                          |

Reviewer: ```suggestion
| **customContent** | No       |      | Allows adding custom content to specific condition types. Each object contains: <br> - `type` (required): The condition type to match <br> - `value` (required): A string, JSONata expression, or a widget definition to render custom content <br> - `header`: Optional header text for the custom content section <br><br> The custom content is rendered above the condition's default message.                                                                                                                                                                                                                                                                                                          |
```
